app/src/main/java/vn/mbm/phimp/me/gallery/PhimpMeGallery.java

```diff
@@ -353,10 +355,27 @@ public void getPopUpData(File file){
             ExifInterface exif_data = new ExifInterface(file.getAbsolutePath());
             Date lastModDate = new Date(file.lastModified());
             long length = file.length()/1024;
+            String img_width = exif_data.getAttribute(ExifInterface.TAG_IMAGE_WIDTH);
+            String img_height = exif_data.getAttribute(ExifInterface.TAG_IMAGE_LENGTH);
+            if(img_width.isEmpty() || img_height.isEmpty() || img_width.equals("0") || img_height.equals("0") ) {
```

Comment:
Position literals first in comparisons, if the second argument is null then NullPointerExceptions can be avoided, they will just return false.